keywords/MobileRestClient.py

```diff
@@ -1715,15 +1734,33 @@ def add_design_doc(self, url, db, name, doc):
         log_r(resp)
         resp.raise_for_status()
 
-        resp_obj = resp.json()
-
-        return resp_obj["id"]
+        # Only return a response if adding to the listener
+        # Sync Gateway does not return a response
+        if self.get_server_type(url) == ServerType.listener:
+            resp_obj = resp.json()
+            return resp_obj["id"]
```

Comment:
Is it possible for a SG test to call this function? If so, will it panic upon no return value?